code-challenges.js

```diff
@@ -20,49 +20,86 @@ const fruit4 = "kiwi"
 
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
-// Pseudo code:
+// Pseudo code: I will input the variable of the boilingpoint water into an if and else if statement to return the following statements depending on the water temperature.
+// My Code #1 
+const boilingPoint = (water) => {
+    if (temp1 > 212) { return "is above boiling point"
+   } 
+     else if (temp1 < 212) {return `${temp1} is below boiling point`} 
+     else if (temp1 == 212){return "is at boiling point"}
+     else return "Check water again"
+   }
+   
+   const temp1 = 42
+   console.log(boilingPoint(temp1))
 
-const temp1 = 42
 // Expected output: "42 is below boiling point"
-
+// My Code #2
+const boiling= (water) => {
+    if (temp2 > 212){return `${temp2} is above boiling point`}
+     else if  (temp2 < 212){return `${temp2} is below boiling point`} 
+     else if (temp2 == 212){return `${temp2} is at boiling point`}
+     else return "Check water again"
+}
 const temp2 = 350
+console.log(boiling(temp2))
 // Expected output: "350 is above boiling point"
-
+// My Code #3
+const boiled= (water) => {
+    if (temp3 > 212){return `${temp3} is above boiling point`}
+     else if  (temp3 < 212){return `${temp3} is below boiling point`} 
+     else if (temp3 == 212){return `${temp3} is at boiling point`}
+     else return "Check water again"
+}
 const temp3 = 212
+console.log(boiled(temp3))
 // Expected output: "212 is at boiling point"
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-// Pseudo code:
+// Pseudo code: Using the .concat function to bring the two arrays together then checking for it's length.
+
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+var padres = (padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns))
+console.log(padres.length)
 // Expected output: 9
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
-// Pseudo code:
+// Pseudo code: Using the split, reverse and join function to flip the string.
```

Comment:
This is a great example of where you could be more descriptive with your pseudocode. Talking about why you are using split. Mention that you need to convert the string into an array before you can use .reverse etc.